dev/drivers/scripts/prep/glwu/jevs_glwu_wave_grid2obs_prep.sh

```diff
@@ -65,10 +74,17 @@ export SITE=$(cat /etc/cluster_name)
 ############################################################
 # CALL executable job script here
 #############################################################
+<<<<<<< HEAD
+$HOMEevs/jobs/JEVS_GLWU_WAVE_GRID2OBS_PREP
+
+#######################################################################
+# Purpose: This does the prep work for the NFCENS wave model
+=======
 $HOMEevs/jobs/JEVS_GLWU_PREP
 
 #######################################################################
 # Purpose: This does the prep work for the GLWU wave model
+>>>>>>> develop
 # Author: Samira Ardani (samira.ardani@NOAA.gov)
 #######################################################################
```

Comment:
@SamiraArdani-NOAA You have the merge tags in this file as well.  Please clean up these files by removing the tag.  

Also - why is this file even here?  If this is the nwps PR, you shouldn't have any changes to the glwu file.  Please remove this file from the PR.